heartbeat/docs/fields.asciidoc

```diff
@@ -9570,7 +9570,7 @@ type: long
 
 --
 
-[[exported-fields-socks5]]
+[[exported-fields-socks6]]
```

Comment:
socks6? Where did that change come from?

Reply:
https://github.com/elastic/beats/pull/21874/files#diff-50eba0891861ef591b619127d66ca19a13aa297f220c1f8b44ee063144d198fcR271

Reply:
Good catch, fixed!